### DISCUSSION TITLE
"start_pref_f" not support on DeepMD-kit v2.0.1 gpu cuda 11.3

### DISCUSSION THREAD

**Discussion by Franklalalala** [deepmodeling/deepmd-kit#1197] [Q&A]
I saw this feature in the code, loss function, which is exactly what i'm looking for. Since my data is not from md trajectory, most of the atoms have a extremely low force. Apply this "atom prefernce" feature may greatly improve the force prediction.
I've seen this. 
https://github.com/deepmodeling/deepmd-kit/discussions/358
And checked my data with this 
https://github.com/deepmodeling/deepmd-kit/discussions/709
I've tried this feature on my cpu version, the results looks acceptable. However, I can't run it with GPU version. BUG is: ArgumentKeyError: [at location `loss`] undefined key `start_pref_pf` is not allowed in strict mode.
What can i do to implement this feature with GPU? Lower my version? If so, which version? Thanks!

**Comment by wanghan-iapcm** [comment]
This is a bug in both cpu and gpu version. plz check if you really use v2.0.1 cpu. 

I create an issue from this discussion.

**Reply by Franklalalala** [reply]
My old cpu version is 1.3.3, and couldn't run it under cpu v2.0.1.
